invenio_subjects_mesh/cli.py

```diff
@@ -19,13 +19,16 @@
 
 
 @click.command()
-def main():
-    """Generate new subjects_mesh.yaml file."""
+@click.argument("year")
+def main(**parameters):
```

Comment:
This is one of my favorite little tricks. I remember seeing for the first time in _Practical Object Oriented Design in Ruby_.